lib/api.js

```diff
@@ -766,6 +766,55 @@ module.exports = function(self, options) {
     return callback(null, result);
   };
 
+  // Retrieve all documents where the draft version has been trashed, but the
+  // live version has not. This method will call `self.getEditable` on the found
+  // docs, which means that the callback is called with the same signature
+  // `(null, modified, unmodified, committable)`
+
+  self.getUncommittedTrash = function(req, callback) {
+    var workflowGuids;
+    var untrashedLiveDocIds;
+
+    return async.series([
+      getTrashedDraftDocs,
+      getUntrashedLiveDocs
+    ], function(err) {
+      if (err) {
+        return callback(err);
+      }
+
+      self.getEditable(req, untrashedLiveDocIds, {}, callback);
```

Comment:
I had some trouble with this, since if I simply passed the `callback` from `self.getUncommittedTrash` as the final argument to `async.series` (after the array of functions), then it would receive the results of all functions, not just the final one. Or did I misunderstand what you meant here? You're very free to edit the code in that case!

Reply:
Oops, you're right, in that situation it should be done your way. Still needs a `return` though.